lib/init.php

```diff
@@ -0,0 +1,4 @@
+<?php
+namespace Trendwerk\Smtp;
+
+$theme = new Smtp();
```

Comment:
I'm not sure to be honest. I've adopted this from another plugin. I just changed this.